tests/cases/compiler/jsFileCompilationTypeParameterSyntaxOfClassExpression.ts

```diff
@@ -0,0 +1,3 @@
+// @allowJs: true

```

Comment:
adding `// @noEmit: true` will suppress the unrelated `Cannot write file ...` error